src/test/java/com/dbteam/repository/PersonRepositoryTest.java

```diff
@@ -28,4 +44,13 @@ void findUserByUsername(@Autowired PersonRepository personRepository) {
         assertTrue(actualPerson.isPresent());
         assertEquals(expectedPerson, actualPerson.get());
     }
+
+    @Test
+    public void getPersonsOfGroup() {
+        // when
+        List<Person> personList = personRepository.getDistinctByGroupChatId(10L);
```

Comment:
How is it work if personRepository doesn't have such a method??

Reply:
hehehe, deleted this test ))